scripts/clang-format.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+# Copyright (C) 2021 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause
+
+# Install Intel FPGA SDK for OpenCL Pro Edition
+# https://fpgasoftware.intel.com/
```

Comment:
These two lines are not part of the license header used in the runtime.

Reply:
Thanks for catching that! now removed.